superchain/extra/addresses/goerli/base.json

```diff
@@ -6,5 +6,10 @@
   "L2OutputOracleProxy": "0x2A35891ff30313CcFa6CE88dcf3858bb075A2298",
   "OptimismMintableERC20FactoryProxy": "0xa88530E2DD811363cA3Ef479dBab3C0BF73d90b1",
   "OptimismPortalProxy": "0xe93c8cD0D409341205A592f8c4Ac1A5fe5585cfA",
-  "ProxyAdmin": "0xbc0Fc544736b7d610D9b05F31B182C8154BEf336"
-}
\ No newline at end of file
+  "ProxyAdmin": "0xbc0Fc544736b7d610D9b05F31B182C8154BEf336",
+  "SystemConfigProxy": "0xb15eea247eCE011C68a614e4a77AD648ff495bc1",
+  "ProxyAdminOwner": "0x444b8C3E4eA49cE15A93D96AfA83D421F6049524",
+  "SystemConfigOwner": "0x4C35Ca57616E0d5fD808574772f632D8dA4eadCa",
```

Comment:
The goal of this file to serve as a reference and source of truth, which we verify in the `CheckSecurityConfigs` script? I ask because as of https://github.com/ethereum-optimism/optimism/pull/6555, we can now only define a chain's `SystemConfig` and then discover everything else. Wondering if we can leverage that in this repo (once everyone is on MCP L1 contracts)